### PR TITLE
Write about Normandy's migration to Auth0

### PR DESCRIPTION
**Pull request by mythmon**
@willkg Does this answer the questions you had?

### CODE REVIEW

File: content/posts/2017-05-03-normandy-and-auth0.mkd [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+changes [in this pull request][normandy#569].
+
+
+# Changes to Nginx
+
+To send this header to Normandy, we modified the Nginx configurations to
+perform authentication via OpenIDC with Auth0. That was implemented
+with [lua-resty-openidc][].
+
+When a request comes in to Nginx, it checks if the request has cookies
```
Comment (willkg) [inline content/posts/2017-05-03-normandy-and-auth0.mkd]: Does this happen for any request at all or is it requests that match the location specified in the .conf file that's covered by OIDC authentication?

Comment (mythmon) [inline content/posts/2017-05-03-normandy-and-auth0.mkd]: I'm not familiar with a .conf file being used. There is a `location` rule that might restrict it to a smaller set of requests, but I don't think we are using that.

You mentioned a testrp repo. I don't think I've seen that, can you send me a link to it?

Comment (willkg) [inline content/posts/2017-05-03-normandy-and-auth0.mkd]: https://github.com/mozilla-iam/testrp.security.allizom.org/tree/master/webserver_configurations/OpenID_Connect/Nginx

I think this is the relevant line in the example:

https://github.com/mozilla-iam/testrp.security.allizom.org/blob/master/webserver_configurations/OpenID_Connect/Nginx/conf.d/social_ldap_pwless.conf#L34

**Comment by willkg** [comment]
This totally answers a bunch of my questions. Thank you!

**Pull request merged**